posthog/hogql/metadata.py

```diff
@@ -53,9 +54,9 @@ def get_hogql_metadata(
             if "mismatched input '<EOF>' expecting" in error:
                 error = "Unexpected end of query"
             response.errors.append(HogQLNotice(message=error, start=e.start, end=e.end))
-        else:
+        elif not settings.DEBUG:
```

Comment:
Why hide this in debug? 🤔

Reply:
It's the opposite – we used to swallow the error always, but we only need to do that in production. In development, we want to fail and get the full traceback